doc/source/kubecluster.rst

```diff
@@ -192,4 +225,45 @@ The pods are created with two default `tolerations <https://kubernetes.io/docs/c
 
 If you have nodes with the corresponding taints, then the worker pods will
 schedule to those nodes (and no other pods will be able to schedule to those
-nodes).
\ No newline at end of file
+nodes).
+
+Scheduler
+~~~~~~~~~
+
+The scheduler can be deployed locally (default) or remotely.  A ``local``
+scheduler is created where the Dask client will be created.
+
+
+.. code-block:: python
+
+   from dask_kubernetes import KubeCluster
+   from dask.distributed import Client
+
+   cluster = KubeCluster.from_yaml('worker-spec.yml', deploy_mode='local')
+   cluster.scale(10)
+   client = Client(cluster)
+
+The scheduler can also be deployed on the kubernetes cluster with
+``deploy_mode=remote``:
+
+
+.. code-block:: python
+
+   from dask_kubernetes import KubeCluster
+   from dask.distributed import Client
+
+   cluster = KubeCluster.from_yaml('worker-spec.yml', deploy_mode='remote')
+   cluster.scale(10)
+   client = Client(cluster)
+
+
+When deploying remotely, the following k8s resources are created:
+
+- A pod with a scheduler running
+- (optional) A pod with a LoadBalancer and complimentary service (svc) to
+  expose scheduler and dashobard ports
+
+By default, the ``scheduler-service-type`` is set to ``ClusterIp``. To
+optionally use the LoadBalancer, change ``scheduler-service-type`` to
+``LoadBalancer``
```

Comment:
Where should folks set this? Looking at the code it seems like it's pulled from `kubernetes.yaml`, which does not seem to be configurable.

Reply:
Is it with `dask.config`?

Reply:
Answered my own question, was with `dask.config.set({"kubernetes.scheduler-service-type": "LoadBalancer"})`, maybe we can add that?

Reply:
Thanks the for the suggestion @holdenk.  Added your thoughts in the last commit